docker/Makefile

```diff
@@ -14,21 +14,21 @@ needslim:
 .PHONY: build-image
```

Comment:
We should probably rename this directory, too.
Maybe `container/` would be more suitable?